lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -493,6 +503,52 @@ spec = do
         r <- joinStakePool ctx (p ^. #id) (w, "Secure Passprase")
         expectResponseCode HTTP.status404 r
 
+    it "STAKE_POOLS_ESTIMATE_FEE_01 - fee matches eventual cost" $ \ctx -> do
+        (_, p:_) <- eventually $
+            unsafeRequest @[ApiStakePool] ctx listStakePoolsEp Empty
+        w <- fixtureWallet ctx
+        fee <- getFromResponse amount <$> joinStakePoolFee ctx (p ^. #id) w
+        r <- joinStakePool ctx (p ^. #id) (w, fixturePassphrase)
+        verify r
+            [ expectFieldEqual amount fee
+            ]
+
+    it "STAKE_POOLS_ESTIMATE_FEE_02 - \
+        \empty wallet cannot estimate fee" $ \ctx -> do
+        (_, p:_) <- eventually $
+            unsafeRequest @[ApiStakePool] ctx listStakePoolsEp Empty
+        w <- emptyWallet ctx
```

Comment:
We are now not testing edge-cases. I think @KtorZ [suggested](https://input-output-rnd.slack.com/archives/GBT05825V/p1575459104040400?thread_ts=1575309301.012300&cid=GBT05825V) that there are existing unit tests for coin selection that should be relevant and better suited for edge-cases. (haven't looked for them myself yet)